common/buildcraft/factory/tile/TileMiner.java

```diff
@@ -92,6 +92,11 @@ public void update() {
     }
 
     @Override
+    public void onLoad() {
+        offset = world.rand.nextInt(10);
```

Comment:
Add 
````java
super.onLoad();
````
Just so that we don't have to worry about future additions to the parent's onLoad function not being called by this.